lib/generators/protokoll/migration/templates/create_custom_auto_increments.rb

```diff
@@ -2,11 +2,13 @@ class CreateCustomAutoIncrements < ActiveRecord::Migration
   def up
     create_table :custom_auto_increments, :force => true do |t|
       t.string  :counter_model_name
+      t.string  :counter_model_scope
```

Comment:
I wonder if the counter by scope should be an add on and have its own migration. That way only the people that actually need the scope would run a `rails g protokoll:migration:scope_by` and the scope migration would be created (or some equivalente generator).

I'm not a fan of pushing this out and having the people not using the scope feature with a blank column in the database.